### ISSUE TITLE
Add to makeOxygen @rdname fun_name

### ISSUE THREAD

**Issue by talgalili** [metrumresearchgroup/sinew#4]
This helps when wanting to reuse the same help in other functions.

**Comment by yonicd** [comment]
This is where the dictionary comes in , I found rdnames and it's similar aliases had some annoying outputs


makeOxygen(tabular,add_fields = c('examples','export'))

#' @title FUNCTION_TITLE
#' @description FUNCTION_DESCRIPTION
#' @param df PARAM_DESCRIPTION
#' @param header PARAM_DESCRIPTION, Default: TRUE
#' @param ... PARAM_DESCRIPTION
#' @return OUTPUT_DESCRIPTION
#' @examples 
#' EXAMPLE1 
#'
#' @export 

dict_loc='https://raw.githubusercontent.com/metrumresearchgroup/sinew/master/man-roxygen/Dictionary-1.R'

makeOxygen(tabular,use_dictionary = dict_loc,add_fields = c('examples','export'))

#' @title FUNCTION_TITLE
#' @description FUNCTION_DESCRIPTION
#' @param df data.frame to convert to table
#' @param header boolean to control if header is created from names(df), Default: TRUE
#' @param ...
#' @return OUTPUT_DESCRIPTION
#' @examples 
#' EXAMPLE1 
#'
#' @export

**Comment by talgalili** [comment]
> I don't understand your answer. 

Also, add hashtag before 
EXAMPLE1
So:
# EXAMPLE1
Or an un attentive user will get errors from it.

**Comment by yonicd** [comment]
the dictionary contains unique values of param from all the functions in a package. (create one with makeDictionary). once you have that (lets say it at a location saved in [dict_loc](https://raw.githubusercontent.com/metrumresearchgroup/sinew/master/man-roxygen/Dictionary-1.R), then makeOxygen will cross reference for you any params that have the same name and add all the other stuff that sinew finds for you automatically.

so this would be a standard output

```r
sinew::makeOxygen(sinew::tabular,add_fields = c('examples','export'))
#' @title FUNCTION_TITLE
#' @description FUNCTION_DESCRIPTION
#' @Param df PARAM_DESCRIPTION
#' @Param header PARAM_DESCRIPTION, Default: TRUE
#' @Param ... PARAM_DESCRIPTION
#' @return OUTPUT_DESCRIPTION
#' @examples
#' #EXAMPLE1
#'
#' @export
```

and this one uses a dictionary

```r
dict_loc='https://raw.githubusercontent.com/metrumresearchgroup/sinew/master/man-roxygen/Dictionary-1.R'

makeOxygen(tabular,use_dictionary = dict_loc,add_fields = c('examples','export'))
#' @title FUNCTION_TITLE
#' @description FUNCTION_DESCRIPTION
#' @Param df data.frame to convert to table
#' @Param header boolean to control if header is created from names(df), Default: TRUE
#' @Param ...
#' @return OUTPUT_DESCRIPTION
#' @examples
#' # EXAMPLE1
#'
#' @export
```

in the regular roxygen framework if you use the `@include` and it has params in it that arent in your header then it adds them as null in the bottom (imo a bug in roxygen)

**Comment by yonicd** [comment]
done. but i still dont like rdname or template

**Comment by talgalili** [comment]
Looking good.
Is it possible to have 
#' @rdname min_depth
#' @export 

Just below 
@title

?

**Comment by yonicd** [comment]
no. it would be a bigger change in the internal code to do that, and it just aesthetics. it doesnt matter where they are for roxygen to run.

**Comment by talgalili** [comment]
ok.
